x/interchainqueries/keeper/keeper.go

```diff
@@ -284,7 +285,8 @@ func (k Keeper) GetClientState(ctx sdk.Context, clientID string) (*tendermintLig
 func (k *Keeper) CollectDeposit(ctx sdk.Context, queryInfo types.RegisteredQuery) error {
 	owner, err := queryInfo.GetOwnerAddress()
 	if err != nil {
-		panic(err.Error())
+		ctx.Logger().Debug("failed to get owner address from given query", "query_info", queryInfo)
```

Comment:
Do you know scenario where Owner will not be valid Address? I think that if we will simply write message to log it may lead to data inconsistency, problem hiding, and this will start growing. Network halting is not the best solution of course, but at least in this case we will see this weird situation if we will catch unexpected error like this.